src/test/components/__snapshots__/ThreadActivityGraph.test.js.snap

```diff
@@ -2521,117 +2521,32 @@ Array [
     "set fillStyle",
     Object {},
   ],
-  Array [
```

Comment:
This was the stack graph part of the SelectedThreadActivityGraph, right?

Reply:
Yes, you're right. It looks like this is also missing from the TrackThread test. Let me add a canvas  snapshot test real quick.

```
  it('matches the 2d canvas draw snapshot', () => {
    const { ctx } = setup(getSamplesProfile());
    expect(ctx.__flushDrawLog()).toMatchSnapshot();
  });
```